src/machine/flash.go

```diff
@@ -133,12 +133,7 @@ func (fl *FlashBuffer) Close() error {
 // You can only seek relative to the start.
 // Also, you cannot use seek before write operations, only read.
 func (fl *FlashBuffer) Seek(offset int64, whence int) (int64, error) {
```

Comment:
This only implements SEEK_START - should at least panic for SEEK_OFFSET and SEEK_END?

Reply:
Done.